### PR TITLE
fixed listview item layout issue

### PR DESCRIPTION
**Pull request by Gammaalpha**
The issue where the text was being cut has been fixed and its been tested on the following emulators.
Pixel
Nexus 5X

### CODE REVIEW

File: app/src/main/res/layout/front_page_listview_item.xml [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
+              android:layout_height="wrap_content"
+              android:background="@drawable/one_sided_border_bottom"
+              android:orientation="horizontal"
+              android:weightSum="1">
 
-        <GridLayout
+    <ImageView
+        android:id="@+id/image"
+        android:layout_width="107dp"
+        android:layout_height="125dp"
```
Comment (davidhuang1550) [inline app/src/main/res/layout/front_page_listview_item.xml]: why not have this match parent or make the parent 125dp

File: app/src/main/res/layout/front_page_listview_item.xml [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
-    >
+              xmlns:app="http://schemas.android.com/apk/res-auto"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:background="@drawable/one_sided_border_bottom"
+              android:orientation="horizontal"
+              android:weightSum="1">
 
-        <GridLayout
+    <ImageView
```
Comment (davidhuang1550) [inline app/src/main/res/layout/front_page_listview_item.xml]: indentation

File: app/src/main/res/layout/front_page_listview_item.xml [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
+        android:id="@+id/image"
+        android:layout_width="107dp"
+        android:layout_height="125dp"
+        android:layout_rowSpan="3"
+        />
+
+    <LinearLayout
```
Comment (davidhuang1550) [inline app/src/main/res/layout/front_page_listview_item.xml]: indentation

**Comment by Gammaalpha** [comment]
Check now if this is what you wanted for the image height.

**Pull request merged**